test/cri-containerd/main.go

```diff
@@ -195,3 +198,59 @@ func findTestNvidiaGPUDevice() (string, error) {
 	}
 	return strings.TrimSpace(string(out)), nil
 }
+
+// Checks to see if the user has the New-CredentialSpec cmdlet installed.
+func hasCmdlet() error {
+	cmdLet := "New-CredentialSpec"
+	input := fmt.Sprintf("Get-Command %s", cmdLet)
+	_, err := exec.Command(
+		"powershell",
+		input,
+	).Output()
+	if err != nil {
+		return fmt.Errorf("necessary cmdlet not installed. Please run 'Install-Module %s'", cmdLet)
+	}
+	return nil
+}
+
+// Generates a gmsa credential spec and places it at 'path'. Dir has to exist it will not be created by
+// New-CredentialSpec. Could extend to os.Stat and if it doesn't exist could mkdir but will just use ContainerPlat
+// dir for now.
```

Comment:
I think it would be better to use a temp dir/temp file created in the test for this. That way we can clean up any state/files we create in a test. 

https://golang.org/pkg/os/#TempDir

Reply:
Good idea :)